### ISSUE TITLE
no help tags / doc.txt file 

### ISSUE THREAD

**Issue by bhurlow** [bhurlow/vim-parinfer#2]

**Comment by chadhs** [comment]
great work. docs would rock, one thing i'm curious about is if this can be used to replace "wrap with" in paredit.  :-)

**Comment by bhurlow** [comment]
@chadhs I'm only mildly familiar with paredit as I'm firmly into vim. Was using vim-sexp instead. Can you briefly explain what paredit "wrap with" does? It like like slurp/barf?

**Comment by chadhs** [comment]
if you had a function and then you wanted function to be nested in an outer form.

one example would be if you suddenly added a middleware layer to a ring based web app.  in the paredit world you could do something like `,w(` and that entire expression would be wrapped in parens and you could enter insert mode right after the new opening `(`.

I suppose you could also start writing the outer form and then do a forward-slurp with paredit or vim-sexp but not sure how one would achieve that with parifiner.

hope that adds some clarity, if not let me know.  :-)

**Comment by bhurlow** [comment]
@chadhs I've found the idiomatic way to do this with parinfer is to simply navigate to the beginning of the form, enter insert mode, insert a `(`, and finally exit insert mode. By existing insert mode, vim-parinfer will send the form to parinfer to be evaluated and the return value will have the other parenthesis automatically balanced.

Also another bonus with parinfer is that we can wrap forms together by using indentation. I've found it pretty easy to `<Tab>` forms around to get them nested how I need them. Of course this assumes the forms will be on separate lines.

I'm trying not to add much functionality beyond whats provided by parinfer itself. However, if parinfer becomes a staple I think I'll need a few things like the wrap you mentioned.

Cheers
